bootstrap.ps1

```diff
@@ -21,8 +21,13 @@ Pop-Location
 
 if (-not (Test-Path Env:/AGENT_OS)) {                                    # If not CI build, i.e. local build (if AGENT_OS envvar is not defined)
     if ($Env:ENABLE_NATIVE -ne "false") {
-        Write-Host "Build release flavor of the native simulator"
         $Env:BUILD_CONFIGURATION = "Release"
+        Write-Host "Build release flavor of the Sparse Simulator"
+        Push-Location (Join-Path $PSScriptRoot "src/Simulation/Simulators/SparseSimulator")
+            .\build.ps1
+        Pop-Location
+
+        Write-Host "Build release flavor of the native simulator"
```

Comment:
I suggest you change it to "full state".

Reply:
Changed.